packages/adminrouter/extra/src/includes/metrics-server-level.conf

```diff
@@ -0,0 +1,18 @@
+# Output fine-grained server directive level metrics using the VTS module.
+#
+# Caution:
+# A request which matches a location that has a VTS module filter defined will be
+# counted on the Nginx location directive level filter but it will not be counted
+# on a filter defined on the server directive level that it matches as well.
```

Comment:
It would be nice to explain how this works in an overall comment somewhere - i.e. what is the relationship and priority for http/location/server filters. Not sure where is best though.

Reply:
I'm writing up a google doc where I can add a section for that.